package.json

```diff
@@ -10,7 +10,7 @@
     "server-start": "node -r dotenv/config server/app.js"
   },
   "dependencies": {
-    "bcrypt": "^3.0.7",
+    "bcryptjs": "^2.4.3",
```

Comment:
This package hasn't been updated since 3 years. So, we need to look out for `bcrypt` installation fix and add it back again once it starts working.

Reply:
Ok